test/fixtures/tricky_doors_payload_injector.ps1

```
@@ -0,0 +1,537 @@
+# LilithOS Tricky Doors Payload Injector
+# Advanced hardware lock bypass system for Nintendo Switch
+
+param(
+    [string]$Drive = "O:",
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'Drive' has been declared but not used.

(PSReviewUnusedParameter)